src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -902,19 +887,8 @@ export default {
 
 	methods: {
 		t,
-		formattedTime,
-
-		updateUserNameNeedsTitle() {
-			// check if ellipsized
-			const e = this.$refs.userName
-			this.isUserNameTitleVisible = (e && e.offsetWidth < e.scrollWidth)
-		},
 
-		updateStatusNeedsTitle() {
-			// check if ellipsized
-			const e = this.$refs.statusMessage
-			this.isStatusTitleVisible = (e && e.offsetWidth < e.scrollWidth)
-		},
```

Comment:
I thought this is intentional by design.

Reply:
Yes, but it doesn't work anymore at ListItem (maybe because of pointer-events disabled somewhere)
Regardless, we got rid of this complex logic at conversation items earlier (also by changing v-tooltip -> title), so it seems right to me to drop it here as well